domains/auth/chart/values.yaml

```diff
@@ -0,0 +1,14 @@
+api:
+  replicaCount: 3
+  image:
+    name: ghcr.io/energinet-datahub/eo-auth
+    tag: 0.0.1
+  env:
+    INTERNALTOKENSECRET
+    TOKENEXPIRYTIME
+    SCOPE
+    AMRVALUES
+    OIDCURL
+    OIDCCLIENTID
+    OIDCCLIENTSECRET
+    BASEURL
```

Comment:
```suggestion
```

these should be removed. setting ENV this way, does not enable us to override them using configmaps from eo-base-environment.